aerolyzer/test_location.py

```diff
@@ -0,0 +1,94 @@
+from retrieve_image_data import RtrvData as Data
+import location
+import image_restriction_main
+
+def test_location_all():
+        filepath = input("Input File Path for Test Image: ")
+        testspassed  = 0
+        if(test_get_coord(filepath)==0):
+                print "test_get_coord passed\n"
+                testspassed+=1
+        else:
+                print "test_get_coord failed\n"
+
+        if(test_coord_to_zip(filepath)==0):
+                print "test_coord_to_zip passed\n"
+                testspassed+=1
+        else:
+                print "test_coord_to_zip failed\n"
+
+        if(test_zip_to_coord(filepath)==0):
+                print "test_zip_to_coord passed\n"
+                testspassed+=1
+        else:
+                print "test_zip_to_coord failed\n"
+
+        if(test_sun_position(filepath)==0):
+                print "test_sun_position passed\n"
+                testspassed+=1
+        else:
+                print "test_sun_position failed\n"
+
+        print "Number of Tests passed "+ str(testspassed) +"/4\n"
+
+
+def test_get_coord(filename):
+        if(image_restriction_main.check_image(filename)):
+                imagedata = Data(filename)
+                exifData = imagedata.get_exif(filename, True, False)
+                coords = location.get_coord(exifData)
+                print "\tOriginal image data: ("+exifData['gps gpslatitude'][1:-1]+","+exifData['gps gpslongitude'][1:-1]+")\n"
+                print "\tConverted image data: ("+str(coords[0])+","+str(coords[1])+")\n"
+                return 0
```

Comment:
@RossDan96 return 1 is reserved for True, and return 0 is reserved for False. See [here](https://www.programiz.com/python-programming/keyword-list) for more details. Update all in this PR accordingly please.